core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -173,14 +174,13 @@ oppia.directive('questionsList', [
           };
 
           ctrl.createQuestion = function() {
+            ctrl.newQuestionSkillIds = [];
+            var selectDifficulty = false;
```

Comment:
Typically we don't start varnames with verbs unless they are functions/methods. Maybe call this `difficultyIsSelected`? (Or `difficultyIsSelectable`, or something ... I'm not actually sure what this variable represents TBH.)

Reply:
This is to indicate if we are in the selectDifficultyView, how about inSelectDifficultyView?

Reply:
Changed it to inSelectDifficultyView

Reply:
Ah, shame it starts with "select". I figured something like XXXViewIsSelected but selectDifficultyViewIsSelected sounds weird :)

Is this a "mode"? If so, then something like `var currentMode = MODE_SELECT_DIFFICULTY` might be better.

Reply:
Done